### PR TITLE
Provide sys description and sys OID

### PR DESCRIPTION
**Pull request by JoProvost**
This commit introduce the possibility to define read only PDU properties as
sysDesc or sysObjectID.

Reference: http://www.oid-info.com/get/1.3.6.1.2.1.1

### CODE REVIEW

File: virtualpdu/pdu/__init__.py [hunk 1]
```diff
@@ -19,6 +19,43 @@
...
+# contain printable ASCII characters.
+sysDescr = (1, 3, 6, 1, 2, 1, 1, 1)
+
+# The vendor's authoritative identification of the network management
+# subsystem contained in the entity. This value is allocated within the SMI
+# enterprises subtree (1.3.6.1.4.1) and provides an easy and unambiguous
+# means for determining 'what kind of box' is being managed. For example, if
+# vendor `Flintstones, Inc.' was assigned the subtree 1.3.6.1.4.1.4242, it
+# could assign the identifier 1.3.6.1.4.1.4242.1.1 to its 'Fred Router'.
+sysObjectID = (1, 3, 6, 1, 2, 1, 1, 2)
```
Comment (fbouliane) [inline virtualpdu/pdu/__init__.py:27]: Above we talk about a 1.3.6.1.4.1 vendor authoritative identification.
This ID is not related to the one described above (1.3.6.1.2.1)

Comment (JoProvost) [reply virtualpdu/pdu/__init__.py:27]: This is directly what we can read here: http://oid-info.com/get/1.3.6.1.2.1.1.2
We could simply remove the comments and trust the reader of this code to read the proper MIB :)

Comment (fbouliane) [reply virtualpdu/pdu/__init__.py:27]: After discussion, we should remove the descriptions because they are a little misleading and the person can query the oid-info to get information on those OID.

**Review by fbouliane — APPROVED** [review]
